service/src/test/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiControllerAiNotebookTest.java

```diff
@@ -116,6 +120,48 @@ public void createAiNotebookInstance_correctZone() throws Exception {
         List.of(WorkspaceConstants.Properties.DEFAULT_RESOURCE_LOCATION));
   }
 
+  @Test
```

Comment:
I don't see a test that validates the notebook state checking. Is there one?

Could be added in here or in the integration tests, but seems like it would be good to cover the error case of the notebook not stopped.